pkg/identity/identity.go

```diff
@@ -475,7 +409,7 @@ func (h *identityHandler) GetToken(certPEM, keyPEM []byte) (roletokens [](*RoleT
 	// The intermediate certificates may be different between each ZTS.
 	// Therefore, ZTS Client for PostRoleCertificateRequest must share the same endpoint as PostInstanceRegisterInformation/PostInstanceRefreshInformation
 	roleClient := zts.NewClient(h.config.Endpoint, t)
-	expireTimeMs := int32(DEFAULT_TOKEN_EXPIRY_TIME_INT * 60)
+	expireTimeMs := int32(config.DEFAULT_TOKEN_EXPIRY_TIME_INT * 60)
```

Comment:
fix it.

Reply:
This logic is removed in later PR. Just ignore is OK.